composer.json

```diff
@@ -22,7 +22,8 @@
     "require": {
         "php": "^8.1",
         "nesbot/carbon": "^2.72.1",
-        "ext-calendar": "*"
+        "ext-calendar": "*",
+        "geniusts/hijri-dates": "^1.1"
```

Comment:
I prefer to not use this package:
https://github.com/spatie/holidays/pull/155#issuecomment-1904467312

You can have a look at Turkey and how they solved Islamic holidays:
https://github.com/spatie/holidays/blob/main/src/Countries/Turkey.php#L11

Reply:
Can I extend the Turkey class to avoid repeating the long list of Islamic dates?

Reply:
As far as I understood @thecaliskan, this is different for most countries, so not really copy pastable.